File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.*;

class Solution {
    public ArrayList<Integer> solution(int[] array, int[][] commands) {
        ArrayList<Integer> answer = new ArrayList<>();
        
        for (int[] command : commands) {
            int[] temp_arr = Arrays.copyOfRange(array, command[0]-1, command[1]);
            Arrays.sort(temp_arr);
            answer.add(temp_arr[command[2]-1]);
        }
        
        return answer;
    }
}

File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for a in range (len(commands)):
        i = commands[a][0]
        j = commands[a][1]
        k = commands[a][2]
        
        arr = array[i-1:j]
        arr.sort()
        answer.append(arr[k-1])
    
    return answer